src/Microsoft.AspNetCore.SignalR.Common/IHubProxy.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public interface IHubProxy
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="methodName"></param>
+        /// <param name="args"></param>
+        Task Invoke(string methodName, params object[] args);
```

Comment:
This is a non blocking invocation. The naming is bad, the Task result tells you when it was sent but doesn't tell you anything else. I started off with void but the name is just confusing.